### PR TITLE
[release/7.0] Update arm queues

### PR DESCRIPTION
**Pull request by sbomer**
Updating to the latest versions per the suggestion in https://github.com/dotnet/runtime/issues/81619#issuecomment-1419450250. I am using the current latest versions from https://github.com/dotnet/versions, instead of floating versions, to match what we do elsewhere. I am not sure that we would want to use floating versions, especially in our servicing branch.

### CODE REVIEW

**Comment by carlossanlop** [comment]
So I merged this fix a couple of hours ago: https://github.com/dotnet/runtime/pull/81545

It changes the non-existent win10 arm64 helix queue for win11 arm64. I noticed this PR was affected by those non-existent 
queues. I also noticed I missed changing the other one, so I'll submit the PR soon. Maybe we should re-run the CI in this PR after that change is merged, to ensure whatever this PR is fixing is properly tested.

cc @hoyosjs

**Comment by carlossanlop** [comment]
Merged the arm64 Windows queue updates. Closing and reopening this PR.

**Comment by agocke** [comment]
Using the floating version seems fine -- I think we're going to be pushed onto that eventually anyway. I would prefer if we had fixed versions and then a system for updating them manually, but that doesn't sound like something that's being invested in.

**Comment by carlossanlop** [comment]
Nice, the CI is almost fully green. There's only one queue that might be stuck (and unfortunately, it is arm64). But I'm glad that the main problems got fixed.

By the way, @cston / @adamsitnik / @Jozkee, it seems this would fix https://github.com/dotnet/runtime/issues/81619 https://github.com/dotnet/runtime/issues/81391 and https://github.com/dotnet/runtime/issues/81544

**Review by carlossanlop — APPROVED** [review]
Let's get this baby merged.

File: src/coreclr/scripts/superpmi_collect_setup.py [hunk 1]
```diff
@@ -403,9 +403,9 @@ def main(main_args):
         helix_queue = "Windows.10.Arm64" if arch == "arm64" else "Windows.10.Amd64.X86.Rt"
     elif platform_name == "linux":
         if arch == "arm":
-            helix_queue = "(Ubuntu.1804.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-bfcd90a-20200121150440"
+            helix_queue = "(Ubuntu.1804.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-20230201170255-8b7d579"
         elif arch == "arm64":
-            helix_queue = "(Ubuntu.1804.Arm64)Ubuntu.1804.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-20210531091519-97d8652"
+            helix_queue = "(Ubuntu.1804.Arm64)Ubuntu.1804.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-20230201170341-8b7d579"
```
Comment (MattGal) [inline src/coreclr/scripts/superpmi_collect_setup.py:408]: @mthalman should these be on the new "floating" versions of the images?

Comment (mthalman) [reply src/coreclr/scripts/superpmi_collect_setup.py:408]: Yes, this should just be updated to `(Ubuntu.1804.Arm64)Ubuntu.1804.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8` unless there's a specific reason it needs to be pinned to this version of the image.

**Pull request merged**